ocular/src/chain/client/automated_tx_handler.rs

```diff
@@ -0,0 +1,292 @@
+#![warn(unused_qualifications)]
+#![allow(unused_imports)]
+// Clippy broken; doesn't recognize certain imports are used and sees them as unused
+
+use crate::{
+    chain::{
+        client::tx::{Account, TxMetadata},
+        config::ChainClientConfig,
+    },
+    error::{AutomatedTxHandlerError, TxError},
+    keyring::Keyring,
+};
+use bip32::PrivateKey;
+use cosmrs::{
+    crypto::{secp256k1::SigningKey, PublicKey},
+    rpc, AccountId, Coin,
+};
+use serde::{Deserialize, Serialize};
+use signatory::{pkcs8::der::Document, pkcs8::LineEnding};
+use std::{fs, path::Path, str::FromStr};
+use tendermint_rpc::endpoint::broadcast::tx_commit::Response;
+
+use super::ChainClient;
+
+// Toml structs
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct NonDelegatedToml<'a> {
+    pub sender: NonDelegatedSender<'a>,
+
+    #[serde(borrow)]
+    pub transactions: Vec<Transaction<'a>>,
+}
+
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct NonDelegatedSender<'a> {
+    pub prefix: &'a str,
+
+    pub source_private_key_path: &'a str,
+
+    pub chain_id: &'a str,
+}
+
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct Transaction<'a> {
+    pub name: &'a str,
+
+    pub destination_account: &'a str,
+
+    pub amount: u64,
+
+    pub denom: &'a str,
+
+    pub account_number: u64,
+
+    pub sequence_number: u64,
```

Comment:
These will be queryable soon via `Account` or some other part of account management

Reply:
Awesome, that'll be sweet.